### PR TITLE
chore: ignore inapplicable snyk report VSCODE-520

### PR DESCRIPTION
**Pull request by addaleax**
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -12,7 +12,12 @@ ignore:
         created: 2023-07-12T09:00:55.557Z
   SNYK-JS-AXIOS-6032459:
     - '*':
-        reason: "This vulnerability doesn't affect us"
+        reason: This vulnerability doesn't affect us
```
Comment (addaleax) [inline .snyk:15]: Autoformat 🤷‍♀️

**Review by himanshusinghs — APPROVED** [review]

**Comment by himanshusinghs** [comment]
Just noticed that socks@2.7.3 is also available now without any dep on ip package. Probably doesn't change much for us except one less snyk ignore, right?

**Comment by addaleax** [comment]
Ah yeah, even better! I'll close this one then

**Comment by addaleax** [comment]
https://github.com/mongodb-js/vscode/pull/693

**Pull request closed without merging**